rpcs3/evdev_joystick_handler.cpp

```diff
@@ -413,10 +413,51 @@ void evdev_joystick_handler::ThreadProc()
             break;
         }
         case EV_ABS:
-            LOG_NOTICE(GENERAL, "Joystick #%d EV_ABS: %d %d", i, evt.code, evt.value);
+        {
 
-            if (evt.code >= ABS_HAT0X && evt.code <= ABS_HAT3Y) {
-                int hat = evt.code - ABS_HAT0X;
+            LOG_NOTICE(GENERAL, "Joystick #%d EV_ABS: %d %d", i, evt.code, evt.value);
+            //duplicate the evt.code and evt.value, so that they're still available
+            //for the analog input
+            int code2=evt.code;
```

Comment:
`int code2 = evt.code;`
instead of 
`int code2=evt.code;`
please. Same with the rest of the file.